server/utils/utils.ts

```diff
@@ -76,3 +76,7 @@ export const convertDateAndTimeInputsToIsoString = <K extends string | number>(
 
   return dateInputObj
 }
+
+export const formatBanner = (copy: string) => {
+  return `<h3 class="govuk-!-margin-top-2">${copy}</h3>`
```

Comment:
I'm a bit wary about us including presentation markup in code that gets used in the controller. Maybe a better way would be to add the `<h3>` in the call to `govukNotificationBanner` in the `_messages` partial. Maybe something like this?

```
{% if successMessages %}
  {% for message in successMessages %}
    {{ govukNotificationBanner({
      html: '<h3 class="govuk-!-margin-top-2">' + message + '</h3>',
      type: 'success'
    }) }}
  {% endfor %}
{% endif %}
```

That way we don't have to remember to user `formatBanner` everywhere too.

Reply:
Another great idea 💡